src/OrchardCore.Modules/OrchardCore.OpenId/Manifest.cs

```diff
@@ -20,7 +20,7 @@
     Name = "OpenID Client",
     Category = "OpenID Connect",
     Description = "Authenticates users from an external OpenID Connect identity provider.",
-    Dependencies = new[] { OpenIdConstants.Features.Core }
+    Dependencies = new[] { OpenIdConstants.Features.Core, "OrchardCore.Users" }
```

Comment:
Erf, why this change? Decoupling the OpenID module from Users was deliberate and reintroducing the coupling is not something we should be doing.

Reply:
@jptissot @deanmarcussen just for infos, if the added dependency is just to register / configure things after, could be adjusted with the startup `Order` / `ConfigureOrder` properties.

Reply:
Yes that was the only reason, I was not aware of the Order / ConfigureOrder properties, thanks.